modules/gcp/rules.tf

```diff
@@ -0,0 +1,135 @@
+locals {
+  internal_network_blocks        = ["10.0.0.0/8"]
+  bitmovin_static_network_blocks = ["104.199.97.13/32", "35.205.157.162/32"]
+}
+
+# enable Compute Engine API
+resource "google_project_service" "compute-engine" {
+  project = var.project_id
+  service = "compute.googleapis.com"
+}
+
+# create auto-mode VPC
+resource "google_compute_network" "bitmovin_vpc_network" {
+  name = var.network_name
+
+  depends_on = [google_project_service.compute-engine]
+}
+
+resource "google_compute_firewall" "bitmovin_allow_internal" {
+  description   = "For communication between the session manager VM instance and its instance manager VM instances"
+  name          = "${google_compute_network.bitmovin_vpc_network.name}-allow-internal"
+  network       = google_compute_network.bitmovin_vpc_network.name
+  source_ranges = local.internal_network_blocks
```

Comment:
you could also take this from the `google_compute_network` data source and also get rid of the local for that